server/_core/webhookHandlers.ts

```
@@ -334,28 +334,14 @@ async function handleEntry(
+  markResponseSentFromOutcome: (
     outcome: MessengerSendOutcome | undefined
-  ) => {
-    if (outcome?.sent) {
-      markResponseSent();
-    }
-  };
-  const trackedCtx: HandlerContext = {
+  ) => void
+): HandlerContext {
+  let trackedCtx: HandlerContext;
+  trackedCtx = {
```

ESLint

[error] 'trackedCtx' is never reassigned. Use 'const' instead.

(prefer-const)